client/src/App.tsx

```diff
@@ -180,7 +182,12 @@ function App() {
 			};
 			setGifs((gifs) => gifs.concat(newGif));
 		});
-	}, []);
+  }, []);
+  
+  const changeBackground = useCallback((backgroundName: string) => {
+    let backgroundImg = backgrounds[backgroundName];
+    document.body.style.backgroundImage = `url(${backgroundImg})`;
```

Comment:
I'd prefer if you set the style for the app class instead of modifying the DOM directly
such as <div className="app" style={{background: backgroundState}}

it helps with styling for instance if we wanted to go to a new page, that page would also be affected by the background which we necessarily wouldn't want

Ideally background should be kept as a state variable, const [background, setBackground] = useState<string>()